storage_integration_test.go

```diff
@@ -0,0 +1,115 @@
+// +build integration
+
+package tprstorage
+
+/*
+	Usage:
+
+		go test -tags=integration $(glide novendor) [FLAGS]
+
+	Flags:
+
+		-integration.ca string
+			CA file path (default "$HOME/.minikube/ca.crt")
+		-integration.crt string
+			certificate file path (default "$HOME/.minikube/apiserver.crt")
+		-integration.key string
+			key file path (default "$HOME/.minikube/apiserver.key")
+		-integration.server string
+			Kubernetes API server address (default "https://192.168.64.16:8443")
+*/
+
+import (
+	"context"
+	"flag"
+	"os/user"
+	"path"
+	"testing"
+
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/microstorage/storagetest"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+var (
+	server  string
+	crtFile string
+	keyFile string
+	caFile  string
+)
+
+func init() {
+	u, err := user.Current()
+	homePath := func(relativePath string) string {
+		if err != nil {
+			return ""
+		}
+		return path.Join(u.HomeDir, relativePath)
+	}
+
+	flag.StringVar(&server, "integration.server", "https://192.168.64.16:8443", "Kubernetes API server address")
```

Comment:
I tried that and:

```
$ go test -tags=integration $(glide novendor) -integration.server http://127.0.0.1:8080
--- FAIL: TestIntegration (0.00s)
        storage_integration_test.go:111: error booting storage: [{/Users/pawel/go/src/github.com/giantswarm/tprstorage/storage.go:153: } {/Users/pawel/go/src/github.com/giantswarm/tprstorage/vendor/github.com/giantswarm/operatorkit/tpr/tpr.go:207: creating TPR tpr-storage.giantswarm.io} {/Users/pawel/go/src/github.com/giantswarm/tprstorage/vendor/github.com/giantswarm/operatorkit/tpr/tpr.go:271: creating TPR tpr-storage.giantswarm.io} {Post http://127.0.0.1:8080/apis/extensions/v1beta1/thirdpartyresources: dial tcp 127.0.0.1:8080: getsockopt: connection refused}]
        storage_integration_test.go:104: error cleaning up TPO %s/%s: %#v giantswarm integration-test Delete http://127.0.0.1:8080/apis/giantswarm.io/v1/namespaces/giantswarm/tprstorages/integration-test: dial tcp 127.0.0.1:8080: getsockopt: connection refused
FAIL
FAIL    github.com/giantswarm/tprstorage        0.065s
```

I use xhyve driver, maybe this IP is fancy. Let's iterate on that.